build_locally.sh

```diff
@@ -18,11 +27,22 @@ gem install jekyll
 gem install bundler -v 1.17.3
 bundle install
 
-# Build website using similar script as Travis, but skip htmlproofer because it fails locally:
-bundle exec jekyll build --strict_front_matter
+# Optionally test website using same script as Travis. 
+# When you are developing locally, you may or may not want to run the full set of tests that Travis does
+if have_travis "$@"; then
+  ./.travis.sh
+fi
 
 # Launch website
-echo "Opening http://localhost:4000 in default browser"
+echo
+echo "-------------------------------------"
+echo "-------------------------------------"
+echo "Preparing to open http://localhost:4000"
+echo "Be sure to refresh auto-opened webpage"
+read -p "Press any key to start serving website..."
```

Comment:
Why do you need this verbose message? And why do you wait for a key press?
Don't you want to run the site immediately?

Reply:
This ensures the user remembers they must refresh the page, because initially it will say 404. its hard to put a sleep timer since you don't know how long it will take the server to spin up.

Reply:
As you said this is a developer tool. Do you really need a multi-line alert?
You didn't reply yet, why you need to wait for the keypress. The web server will be launched only after the key press. I think it makes sense to start the server asap :wink:

Reply:
@rhaschke I think you're being really picky about a web developer tool I use for testing pull requests. Yes, I think its easy to miss this message if you don't make it large. 

They key press is to ensure you read the message before the web browser opens. Once the browser opens, the terminal is hidden. The user will be confused because a "page not found" error will be displayed. The user just needs to then refresh the page. 

I've been using this tool for a long time, many times, and it works great. Let's not over analyze something so simple...

Reply:
@davetcoleman, I just wanted to understand your motivation for this change. I used the tool recently and it worked just fine for me as is. A keypress was not required because the server starts faster than my browser. IMHO, this change is not necessary. If you insist, go ahead with it :wink:

Reply:
i've removed it.